agent/php_internal_instrument.c

```diff
@@ -1527,10 +1560,67 @@ NR_INNER_WRAPPER(mysqli_stmt_prepare) {
  */
 NR_INNER_WRAPPER(memcache_function) {
   nr_php_instrument_datastore_operation_call(nr_wrapper, NR_DATASTORE_MEMCACHE,
-                                             nr_wrapper->extra, NULL,
+                                             nr_wrapper->extra, NULL, false,
                                              INTERNAL_FUNCTION_PARAM_PASSTHRU);
 }
 
+NR_INNER_WRAPPER(memcached_connect_function) {
+  char* host = NULL;
+  nr_string_len_t host_len = 0;
+  zend_long port = 0;
```

Comment:
```suggestion
  int zcaught = 0;
  zend_long port = 0;
```

Reply:
fixed in [c2396ae](https://github.com/newrelic/newrelic-php-agent/pull/958/commits/c2396ae73c4e70af513e62416c3e42dd2ac8e6f1)